HowTo/JBOSS/avoid_timeouts_obsolete_nodes.md

```diff
@@ -0,0 +1,29 @@
+# How to: Avoid JBOSS clustering timeouts due to obsolete discovery files
+
+During the normal lifecycle of your JBOSS EAP app there can be multiple events that can trigger your app to move across different nodes on the App Service infrastucture, this can be caused by user opertations, such as deployments, scale operations, etc... or platform maintenance oeprations.

```

Comment:
I think this would read better and fixes some typos: 

During the normal lifecycle of your JBoss EAP web app multiple events that can trigger your app to move across different nodes on the App Service infrastructure. This can be caused by user operations (such as deployments). scale operations and platform maintenance operations.